src/screens/Settings/FollowingFeedPreferences.tsx

```diff
@@ -0,0 +1,143 @@
+import {Bubbles_Stroke2_Corner2_Rounded as BubblesIcon} from '#/components/icons/Bubble'
+import {CloseQuote_Stroke2_Corner1_Rounded as QuoteIcon} from '#/components/icons/Quote'
+import {Repost_Stroke2_Corner2_Rounded as RepostIcon} from '#/components/icons/Repost'
+import * as Layout from '#/components/Layout'
+import * as SettingsList from './components/SettingsList'
+
+type Props = NativeStackScreenProps<
+  CommonNavigatorParams,
+  'PreferencesFollowingFeed'
+>
+export function FollowingFeedPreferencesScreen({}: Props) {
+  const {_} = useLingui()
+
+  const {data: preferences} = usePreferencesQuery()
+  const {mutate: setFeedViewPref, variables} =
+    useSetFeedViewPreferencesMutation()
+
+  const showReplies = !(
+    variables?.hideReplies ?? preferences?.feedViewPrefs?.hideReplies
+  )
+
+  const showReposts = !(
+    variables?.hideReposts ?? preferences?.feedViewPrefs?.hideReposts
+  )
+
+  const showQuotePosts = !(
+    variables?.hideQuotePosts ?? preferences?.feedViewPrefs?.hideQuotePosts
+  )
+
+  const mergeFeedEnabled = Boolean(
+    variables?.lab_mergeFeedEnabled ??
+      preferences?.feedViewPrefs?.lab_mergeFeedEnabled,
+  )
+
+  return (
+    <Layout.Screen testID="followingFeedPreferencesScreen">
+      <Layout.Header title={_(msg`Following Feed Preferences`)} />
+      <Layout.Content>
+        <SettingsList.Container>
+          <SettingsList.Item>
+            <Admonition type="tip" style={[a.flex_1]}>
+              These settings only apply to the Following feed.
+            </Admonition>
+          </SettingsList.Item>
+          <Toggle.Item
+            type="checkbox"
+            name="show-replies"
+            label={_(msg`Show replies`)}
+            value={showReplies}
+            onChange={value =>
+              setFeedViewPref({
+                hideReplies: !value,
+              })
+            }>
+            <SettingsList.Item>
+              <SettingsList.ItemIcon icon={BubblesIcon} />
+              <SettingsList.ItemText>
+                <Trans>Show replies</Trans>
+              </SettingsList.ItemText>
+              <Toggle.Platform />
+            </SettingsList.Item>
+          </Toggle.Item>
+          <Toggle.Item
+            type="checkbox"
+            name="show-reposts"
+            label={_(msg`Show reposts`)}
+            value={showReposts}
+            onChange={value =>
+              setFeedViewPref({
+                hideReposts: !value,
+              })
+            }>
+            <SettingsList.Item>
+              <SettingsList.ItemIcon icon={RepostIcon} />
+              <SettingsList.ItemText>
+                <Trans>Show reposts</Trans>
+              </SettingsList.ItemText>
+              <Toggle.Platform />
+            </SettingsList.Item>
+          </Toggle.Item>
+          <Toggle.Item
+            type="checkbox"
+            name="show-quotes"
+            label={_(msg`Show quotes`)}
```

Comment:
```suggestion
            label={_(msg`Show quote posts`)}
```
Small tweak suggestion: what about aligning the label text with the displayed text?